### PR TITLE
refac: Remove exchange from basis data in settlement report

### PR DESCRIPTION
**Pull request by JesperGomes**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)
- [ ] Documentation has been updated
- [ ] The integration [event catalog](https://energinet.atlassian.net/wiki/spaces/D3/pages/555581556/Event+catalog) has been updated
- [ ] C4 diagrams have been updated and Team Outlaws has been informed about relevant changes

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405071800_Modify_settlement_report_metering_point_time_series_v1_view.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+SELECT m.calculation_id,
+       m.metering_point_id,
+       m.metering_point_type,
+       m.resolution,
+       m.grid_area_code,
+       m.energy_supplier_id,
+       DATE_TRUNC('day', FROM_UTC_TIMESTAMP(t.observation_time, 'Europe/Copenhagen')) AS observation_day,
+       ARRAY_SORT(ARRAY_AGG(struct(t.observation_time, t.quantity)))                  AS quantities
+FROM {BASIS_DATA_DATABASE_NAME}.metering_point_periods AS m
+         INNER JOIN (SELECT calculation_id FROM {BASIS_DATA_DATABASE_NAME}.calculations) AS c ON c.calculation_id = m.calculation_id
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202405071800_Modify_settlement_report_metering_point_time_series_v1_view.sql:14]: Can this be simplified to...?
```suggestion
         INNER JOIN {BASIS_DATA_DATABASE_NAME}.calculations AS c ON c.calculation_id = m.calculation_id
```

**Review by BjarkeMeier — APPROVED** [review]

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405071800_Modify_settlement_report_metering_point_time_series_v1_view.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+GO
+
+CREATE VIEW IF NOT EXISTS {SETTLEMENT_REPORT_DATABASE_NAME}.metering_point_time_series_v1 as
+SELECT m.calculation_id,
+       m.metering_point_id,
+       m.metering_point_type,
+       m.resolution,
+       m.grid_area_code,
+       m.energy_supplier_id,
+       DATE_TRUNC('day', FROM_UTC_TIMESTAMP(t.observation_time, 'Europe/Copenhagen')) AS observation_day,
```
[the remark this answers is not in the document]
Comment (JesperGomes) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405071800_Modify_settlement_report_metering_point_time_series_v1_view.sql:11]: I didn't put much thought into this (my modification to the view didn't involve this). But I guess we could use a variable here instead!?!

[the remark this answers is not in the document]
Comment (JesperGomes) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405071800_Modify_settlement_report_metering_point_time_series_v1_view.sql:11]: Or...maybe it's not that easy - I guess it's not worth dealing with now :-/

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405071800_Modify_settlement_report_metering_point_time_series_v1_view.sql (hunk 1 above)
Comment (JesperGomes) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405071800_Modify_settlement_report_metering_point_time_series_v1_view.sql:14]: should also work

**Pull request closed without merging**